### DISCUSSION TITLE
Parser.tagVariable workaround in webpack 4

### DISCUSSION THREAD

**Discussion by 9-lives** [webpack/webpack#18786] [Q&A] [answered]
@alexander-akait Hi, i saw your answer about Parser.tagVariable on https://github.com/webpack/webpack/issues/12385. 
It works in webpack 5 but there is no such a method in webpack 4. Any workaround?

**Comment by 9-lives** [comment]
I know that it's a way to analyze AST in parser program hook. But i don't wanna parse the AST again as possible as i can.

**Reply by alexander-akait** [reply, marked as answer]
It will not work for v4, you need to update
